FVCalc.java

```diff
@@ -1,6 +1,13 @@
 // Computes the future value of a saving investment.
 public class FVCalc {
 	public static void main(String[] args){
-		// Replace this comment with your code
+		int money = Integer.parseInt(args[0]);
+		double rate = Double.parseDouble(args[1]);
+		int year = Integer.parseInt(args[2]);
+
+		int futureValue = (int)(money * Math.pow(rate / 100 + 1, year));
+
+		System.out.println("After " + year + " years" + "," + " a" +  " $" + money + " saved" + " at " + rate + "%" + " will " + "yield " + "$" + futureValue);
```

Comment:
see styling guidelines,
also could have wrote:
`"After " + year + " years, a $" + money + " saved at " + rate + "% will yield $" + futureValue`
Don't need to split every word into different string